obal/data/playbooks/nightly/nightly.yaml

```diff
@@ -13,21 +13,52 @@
       when: nightly_releaser is defined and '--arg jenkins_job=' in (build_package_tito_releaser_args | join(' '))
 
     - block:
-      - name: 'set nightly_macro_args'
-        set_fact:
-          nightly_macro_args:
-            - "--arg rpmbuild_options=\"--define 'nightly .{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}git{{ nightly_githash[:7] }}'\""
-
-      - name: 'append nightly_macro_args to build_package_tito_releaser_args'
-        set_fact:
-          build_package_tito_releaser_args: "{{ build_package_tito_releaser_args }} + {{ nightly_macro_args }}"
-
-      - name: copy source files
-        copy:
-          src: "{{ item }}"
-          dest: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}/"
-        with_items: "{{ nightly_sourcefiles }}"
+        - name: 'set package_dir'
```

Comment:
we have the same code in `obal/data/roles/setup_sources/tasks/main.yml`, which makes me think this should go to an own role (`get_package_dir` or something) and then be included in both places.